src/utils.js

```diff
@@ -57,12 +57,19 @@ async function transformTrxToJson(filePath) {
 
   if (XMLValidator.validate(xmlData.toString()) === true) {
     const parsedTrx = xmlParser.parse(xmlData);
+
+    // Verify the parsed data contains all the required items so we can fail fast with more descriptive error messages.
+    if (!isParsedTrxValid(parsedTrx, filePath)) {
+      return;
+    }
+
+    // QUESTION:  Why is this an important thing to note?  Is it just leftover from the original implementation?  Can it be removed?
     const runInfos = parsedTrx.TestRun.ResultSummary.RunInfos;
     if (runInfos && runInfos.RunInfo._outcome === 'Failed') {
       core.warning('There is trouble');
     }
```

Comment:
Thanks.  I think I'll take it out.  I was trying to think through whether it would have any negative connotations for users of the action...maybe some people are looking for this?  But I don't think that's the case.  My guess is, it really just is a holdover from the initial implementation.